test/account/PerHookData.t.sol

```diff
@@ -341,13 +344,15 @@ contract PerHookDataTest is CustomValidationTestBase {
             ),
             abi.encode(_counter)
         );
-
+        // patched to also work during SMA tests by differentiating the validation
+        _signerValidation =
+            ModuleEntityLib.pack(address(singleSignerValidationModule), TEST_DEFAULT_VALIDATION_ENTITY_ID - 1);
```

Comment:
Could we pick a different validation entity ID for this test and declare it a constant, instead of deriving it from the test default one? It would help us isolate changes to the default from downstream effects to this test.